### PR TITLE
CA 131 added PropTypes to project

### PR DESCRIPTION
**Pull request by DawsonDease**

### CODE REVIEW

**Review by JaycobDuffel — APPROVED** [review]

File: components/AccountModal.js [hunk 1]
```diff
@@ -30,4 +31,10 @@ const AccountModal = (props) => {
   );
 };
 
+AccountModal.propTypes = {
+  visible: PropTypes.bool,
```
Comment (katiemcgoff) [inline components/AccountModal.js]: is it visible or visibility?

File: components/AppFormField.js [hunk 2]
```diff
@@ -39,4 +39,9 @@ function AppFormField({
   );
 }
 
+AppFormField.propType = {
```
Comment (katiemcgoff) [inline components/AppFormField.js]: propType -> propTypes (with an S)

File: screens/RegisterEmail.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { useContext, useState } from "react";
 import { StyleSheet, View } from "react-native";
 import * as Yup from "yup";
-
+import Proptypes from "prop-types";
```
Comment (katiemcgoff) [inline screens/RegisterEmail.js]: should be camel case (PropTypes)

File: screens/RegisterName.js [hunk 4]
```diff
@@ -46,6 +46,11 @@ export default function RegisterName({ name }) {
   );
 }
 
+RegisterName.propTypes = {
+  name: PropTypes.string,
+  navigatiom: PropTypes.func,
```
Comment (katiemcgoff) [inline screens/RegisterName.js]: I don't think navigation is a prop. Also it's spelled wrong but probably irrelevant.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
a few notes

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**